quakemigrate/signal/local_mag/amplitude.py

```diff
@@ -770,8 +770,14 @@ def _measure_signal_amps(self, amps, tr, windows, method="RMS",
                 _, filter_gain = sosfreqz(filter_sos, worN=[approx_freq],
                                           fs=tr.stats.sampling_rate)
                 filter_gain = np.abs(filter_gain[0])
-                half_amp /= filter_gain
-                average_amp /= filter_gain
+                if not filter_gain:
```

Comment:
What does `np.abs()` of a (presumably) `None` return? Still `None`?

Reply:
See commit message - it returns `0.`, not `None`

Reply:
So the test is using the falsity of `0` - can anything else be that would trigger this (e.g. a `None`) for which we wouldn't want the same message? I can't think of one but just in case.

Reply:
I chose this test to also cover `None` or `NaN` - not immediately obvious how they would come about, but they would also lead the following lines to break. Potentially in that case a different message would be appropriate, but these are hypotheticals of hypotheticals - the printout of the invalid frequency should show if something different is going wrong.

Reply:
Aye, ok, all good.